### PR TITLE
♻️🐛Maintenance increase heap memory to compile the frontend

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

Increasing the heap memory while building the frontend. Probably there is a memory leak.
```bash
#15 ERROR: executor failed running [/bin/sh -c qx compile --target=build   --set-env osparc.vcsOriginUrl="${VCS_URL}"   --set-env osparc.vcsRef="${VCS_REF}"   --set-env osparc.vcsRefClient="${VCS_REF_CLIENT}"   --set-env osparc.vcsStatusClient="${VCS_STATUS_CLIENT}" &&   ls -la build-output]: exit code: 134
------
 > [build-client 7/7] RUN qx compile --target=build   --set-env osparc.vcsOriginUrl="https://github.com/ITISFoundation/osparc-simcore"   --set-env osparc.vcsRef="c2df268"   --set-env osparc.vcsRefClient="c2df268"   --set-env osparc.vcsStatusClient="clean" &&   ls -la build-output:
#15 149.5 
#15 149.5     0: ExitFrame [pc: 0xe24510dbe1d]
#15 149.5     1: StubFrame [pc: 0xe24510d4051]
#15 149.5 Security context: 0x2ba02959e6e9 <JSObject>
#15 149.5     2: split [0x2ba0295906c9](this=0x2[786](https://github.com/ITISFoundation/osparc-simcore/runs/5775244339?check_suite_focus=true#step:9:786)29f06cf9 <String[4]: null>,0x3638c84bf989 <JSRegExp <String[5]: \r?\n>>)
#15 149.5     3: print [0x193b2f1131c9] [0x278629f026f1 <undefined>:~5771] [pc=0xe24528ad151](this=0x193b2f113249 <Object map = 0x1af6ccbeb6b9>,str=0x278629f022b1 <null>)
#15 149.5     4: /* anonymous */(aka /* anonymous */) [0x2f...
#15 149.5 
#15 149.5 FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
#15 155.1 Aborted (core dumped)
```

### CODE REVIEW

File: scripts/hadolint.bash [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-# dockerfile linter tool
-# - https://github.com/hadolint/hadolint
-exec docker run --rm --interactive hadolint/hadolint < "$@"
\ No newline at end of file
+
+docker run --rm -i hadolint/hadolint hadolint \
```
Comment (pcrespov) [inline scripts/hadolint.bash:3]: Q: what is the different w/o ``exec``?

MINOR: use long flag-names i.e.
```
-i, --interactive                    Keep STDIN open even if not attached
```

**Review by pcrespov — APPROVED** [review]
Thanks so much for looking into this!

**Review by odeimaiz — APPROVED** [review]
Thanks!

**Review by GitHK — COMMENTED** [review]
This was an odd one

**Pull request merged**